src/binary-reader-ir.cc

```diff
@@ -868,36 +865,36 @@ Result BinaryReaderIR::OnLocalNameLocalCount(Index index, Index count) {
 }
 
 Result BinaryReaderIR::OnInitExprF32ConstExpr(Index index, uint32_t value) {
-  Expr* expr = Expr::CreateConst(Const(Const::F32(), value, GetLocation()));
+  auto expr = new ConstExpr(Const(Const::F32(), value, GetLocation()));
```

Comment:
Another nit, I generally prefer `auto*` over `auto` when we know it will always be a pointer. But if we are going to go to `unique_ptr` soon, then probably not worth redoing it.